core/trino-main/src/main/java/io/trino/server/SessionPropertyDefaults.java

```diff
@@ -71,11 +71,15 @@ public void addConfigurationManagerFactory(SessionPropertyConfigurationManagerFa
     public void loadConfigurationManager()
             throws IOException
     {
-        File configFile = CONFIG_FILE.getAbsoluteFile();
+        loadConfigurationManager(CONFIG_FILE.getAbsoluteFile());
```

Comment:
`Add test for default session properties in transaction` should be squashed with commit which adds new functinality.

Reply:
Yes. It was like this originally, but i wanted to split it to give author to @findinpath. How it's now.